packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -127,9 +153,15 @@ class BottomSheet extends Component {
 					<View style={ styles.dragIndicator } />
 					{ hideHeader && ( <View style={ styles.emptyHeaderSpace } /> ) }
 					{ ! hideHeader && getHeader() }
-					<View style={ [ styles.content, contentStyle ] }>
+					<ScrollView
+						bounces={ this.state.bounces }
+						onScroll={ this.onScroll }
+						scrollEventThrottle={ 16 }
+						style={ { maxHeight } }
+						contentContainerStyle={ [ styles.content, contentStyle ] }
+					>
 						{ this.props.children }
```

Comment:
One thing which is worrying me ( _and I'm not proud of it :(_ ) is placing a `FlatList` within a `ScrollView`. Generally, it shouldn't have any performance issues since scrolling is disabled in a `FlatList`. There is no issue with rewriting it, but we're going to lose nice features such as `ItemSeparatorComponent` or `numColumns`. Please let me know wdyt.